src/Components/Common/Export.tsx

```diff
@@ -47,8 +47,9 @@ export const ExportMenu = ({
         icon={<CareIcon className="care-l-import" />}
         className="bg-white hover:bg-primary-100 text-primary-500 enabled:border border-primary-500 tooltip"
       >
-        {exportItems.map((item) => (
+        {exportItems.map((item, index) => (
```

Comment:
@mecskyverse prefer uuid for keys over index, can you check if there is item.id or any other unique identifier, if not use index

Reply:
Ok sure I will check for it.